components/frontend/.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "jest": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:react/jsx-runtime"
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "no-unused-vars": [
+            "error",
+            {
+                "argsIgnorePattern": "^_"
+            }
+        ]
+    },
+    "overrides": [
+        {
+            "files": [
+                "*.test.js"
+            ],
+            "rules": {
+                "no-import-assign": "off"
+            }
+        }
+    ],
+    "settings": {
+        "react": {
+            "version": "detect"
```

Comment:
Is it not possible to pin this to the version we've set in package.json, or is that not something easily updated by the auto update PR bots?

Reply:
I don't know whether Dependant can pick up a version number in this config file, but why would we want to pin the React version, assuming the detection works of course?